src/app/containers/FrontPageSection/index.jsx

```diff
@@ -125,7 +183,10 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
       >
         {group.strapline.name}
       </SectionLabel>
-      {items.length > 1 ? (
+      {/* eslint-disable-next-line no-nested-ternary */}
```

Comment:
To avoid this exception and having nested ternary operators, you could abstract the logic for the Story Promo in a separate component as you did with the `UsefulLinksComponent`.

```
const StoryPromoWrapper = ({all the props}) => {
     return items.length > 1 ? (
        <MarginWrapper firstSection={isFirstSection}>
          <StoryPromoUl>
            {items.map((item, index) => (
              <StoryPromoLi key={item.id}>
                <StoryPromoComponent
                  item={item}
                  sectionNumber={sectionNumber}
                  storyNumber={index}
                />
              </StoryPromoLi>
            ))}
          </StoryPromoUl>
        </MarginWrapper>
      ) : (
        <MarginWrapper firstSection={isFirstSection} oneItem>
          <StoryPromoComponent
            item={items[0]}
            sectionNumber={sectionNumber}
            storyNumber={0}
          />
        </MarginWrapper>
     )
};

{group.strapline.name === 'Useful links' ? (
        <UsefulLinksComponent items={items} script={script} service={service} />
 ) : (
       <StoryPromoWrapper `all the props`/> 
)};
```